src/ImageSharp/Color/Color.cs

```diff
@@ -286,14 +287,10 @@ public TPixel ToPixel<TPixel>()
         /// Bulk converts a span of <see cref="Color"/> to a span of a specified <typeparamref name="TPixel"/> type.
         /// </summary>
         /// <typeparam name="TPixel">The pixel type to convert to.</typeparam>
-        /// <param name="configuration">The configuration.</param>
         /// <param name="source">The source color span.</param>
         /// <param name="destination">The destination pixel span.</param>
         [MethodImpl(InliningOptions.ShortMethod)]
-        public static void ToPixel<TPixel>(
-            Configuration configuration,
-            ReadOnlySpan<Color> source,
-            Span<TPixel> destination)
+        public static void ToPixel<TPixel>(ReadOnlySpan<Color> source, Span<TPixel> destination)
```

Comment:
This is a breaking change in the Api? Should we not limit this PR to only remove non breaking changes?

Reply:
I added `Configuration` to the parameter list to enable the optimization of this method by utilizing the `PixelOperations<TPixel>.FromRgba64()` batch conversion method. The abstract `PixelOperations<TPixel>` methods take `Configuration` for the case the implementation decides to refer to `Configuration.MemoryAllocator` to allocate temporary buffers.

If we don't like this pattern, we can open an issue to re-evaluate the design, after carefully analyzing our current conversion methods, which no longer need special logic for .NET Framework, though we may need to account for non-Intel, non-coreclr targets.

But I would avoid making sneaky public API changes in a big refactor PR. Is this the only one ore is there more?

Reply:
Let's re-add it for now then and add a TODO in the code so intent is known. 

>Is this the only one ore is there more?
Some of the methods on `ColorSpaceConverter` are now static. I want to keep them like that though.